website/docs/language/state/encryption.mdx

```diff
@@ -0,0 +1,115 @@
+
+Finally, before enable encryption, please exercise your disaster recovery plan and make a temporary backup of your unencrypted state file. Once you enable encryption, OpenTofu cannot read your state file without the correct key.
+
+## Configuration
+
+You can configure encryption in OpenTofu either by specifying the configuration in the OpenTofu code, or using the `TF_ENCRYPTION` environment variable. Both solutions are equivalent and if you use both, OpenTofu will merge the two configurations, overriding any code-based settings with the environment ones.
+
+The basic configuration structure looks as follows:
+
+<Tabs>
+    <TabItem value="code" label="Code" default>
+        <CodeBlock language={"hcl"}>{ConfigurationTF}</CodeBlock>
+    </TabItem>
+    <TabItem value="env-sh" label="Environment (Linux/UNIX shell)">
+        <CodeBlock language={"shell"}>{ConfigurationSH}</CodeBlock>
+    </TabItem>
+    <TabItem value="env-ps1" label="Environment (Powershell)">
+        <CodeBlock language={"powershell"}>{ConfigurationPS1}</CodeBlock>
+    </TabItem>
+</Tabs>
+
+:::warning
+
+Once your data is encrypted, do not rename key providers and methods in your configuration! The encrypted data contains metadata related to their specific names. Instead, use a [fallback block](#key-and-method-rollover).
+
+:::
+
+:::tip
+
+You can use the [JSON configuration syntax](/docs/language/syntax/json/) instead of HCL for encryption configuration.
+
+:::
+
+:::tip
+
+If you use environment configuration, you can include the following code configuration to prevent unencrypted data from being written in the absence of an environment variable:
+
+<CodeBlock language="hcl">{Enforce}</CodeBlock>
+
+:::
+
+## Key providers
+
+There are currently no supported key providers.
+
+## Methods
+
+### AES-GCM
+
+The only currently supported encryption method is AES-GCM. You can configure it in the following way:
+
+<CodeBlock language="hcl">{AESGCM}</CodeBlock>
+
+:::warning
+
+When using a key-management system as a key provider, always configure automatic key rotation in conjunction with AES-GCM, otherwise your security may be compromised! (This does not affect passphrase-based local keys.)
+
+:::
+
+## Key and method rollover
+
+In some cases, you may want to change your encryption configuration. This can include renaming a key provider or method, changing a passphrase for a key provider, or switching key-management systems. OpenTofu supports an automatic rollover of your encryption configuration if you provide your old configuration in a `fallback` block:
+
+<CodeBlock language="hcl">{Fallback}</CodeBlock>
+
+If OpenTofu fails to read your state or plan file with the new method, it automatically tries the fallback method. When OpenTofu saves your state or plan files, it will always use the new method.

```

Comment:
Maybe we should embolden read and saves here to highlight the difference in actions being the driving factor behind the differences.